source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+    <div class="panel-heading panel-drop-heading">
+    <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#unique-anchor">
+      <h3 class="info panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-wrench"></span> CSR Details</h3>
+    </a>
+    </div>
+    <div id="unique-anchor" class="collapse" markdown="1" style="padding:10px;">
+    ### CSR Details Required {.info}
+
+    When requesting a CSR file, you must provide Pantheon Support with the following information:
+
+     - Common Name (e.g. `example.com`)
+     - DNS Names (e.g. `example.com`:`www.example.com`:`blog.example.com`:`dev.example.com`)
+     - Organization (e.g. `ACME, Inc.`)
+     - Organizational Unit \[optional\] (e.g. `Engineering`)
+     - Country (e.g. `US`)
+     - Locality (e.g. `San Francisco`)
+     - State / Province (e.g. `California`)
+     - Emails addresses \[optional\] (e.g. `someone@example.com`:`admin@example.com`)
+    </div>
+    </div>
+
+2. Pantheon support will provide you with the CSR file, to pass on to your **Certificate Authority** (CA).
+
+3. Once you have a set of certificates from the CA, send us:
+
+    - The end-client certificate
+    - Any intermediate certficates provided by the CA.<br><br>
+
+    Be sure to send these as separate files, not a "chained cert".
+
+4. [Test](#test-before-going-live) your new records using tools like `dig` (optional, but strongly recommended).
+
+5. Update your [DNS](/docs/dns/) records, including `A`, `AAAA`, and [CAA](#add-caa-records) records.
+
+
+### Test Before Going Live
+
+Test production domain(s) before updating DNS by overriding DNS on your local computer from your local `hosts` file:
+
+```bash
+192.0.2.5       example.com
+```
+
+In the example `hosts` file above, replace `192.0.2.5` with the IP address provided by Pantheon in the **<span class="glyphicons glyphicons-global"></span>  Domains / HTTPS** tab of your Site Dashboard, and `example.com` with your domain name.
+
+
+For non-production domains, test on any environment (Dev, Test, Live or Multidev), just make sure to include the non-production domains on your certificate.
+
+### Add CAA Records
+
+A **Certification Authority Authorization** (CAA) record is used to specify which certificate authorities (CAs) are allowed to issue certificates for a domain. In order to ensure your custom certificate is served for all traffic, you must prevent Let’s Encrypt from issuing certificates. You have two options to prevent Let’s Encrypt from issuing certificates for domains on your custom certificate:
+
+ - An empty CAA policy,
+ - CAA records permitting your CA, but not Let’s Encrypt.
+
+To help generate CAA records, please see the free online tool: <https://sslmate.com/caa/>
+
+## Technical Specifications
```

Comment:
Single source this table and include by partial file OR link to the existing content instead of duplciating

Reply:
This table is different from the one on other pages because it:

 - Includes a third column for custom certificates that we don't want to include on other pages,
 - Does not include the first row from other versions.

Reply:
Ah, i missed that. Thanks for clarifying